config/unicorn.rb

```diff
@@ -31,5 +31,7 @@
 end
 
 # ログの出力
-stderr_path File.expand_path('log/unicorn.log', ENV['RAILS_ROOT'])
-stdout_path File.expand_path('log/unicorn.log', ENV['RAILS_ROOT'])
+unless ENV.has_key?('RAILS_LOG_TO_STDOUT')
```

Comment:
__[RuboCop]__

Style/PreferredHashMethods: Use `Hash#key?` instead of `Hash#has_key?`.


<!-- Comment by [SideCI](https://sideci.com) -->